webapp/components/channel_header.jsx

```diff
@@ -362,6 +378,64 @@ export default class ChannelHeader extends React.Component {
                     </ToggleModalButton>
                 </li>
             );
+        } else if (isGroup) {
+            dropdownContents.push(
```

Comment:
Could this be done in a way that we're not duplicating the JSX for these buttons?